### ISSUE TITLE
Default line lenght does not match gdformat default

### ISSUE THREAD

**Issue by chrisl8** [EddieDover/vscode-formatter-godot#10]
As of this PR https://github.com/EddieDover/vscode-formatter-godot/pull/9 the default line length is now 80, but the default built into `gdformat` is 100, so after updating this extension in VSCode all of my files start changing their formatting without me asking for that.

What I would really rather see is putting settings like this into a config file that is committed to the code base, so that every developer who clones the code would use the same settings. Otherwise it is easy for people trying to make PRs to accidentally commit a bunch of formatting changes to your code that you don't want because they didn't edit their VSCode settings.

`gdformat` does have a config file now https://github.com/Scony/godot-gdscript-toolkit/pull/286 so I could ask to add line length to it, however it would **also** be nice to be able to read and update the config file from the settings menu. Does that even make sense?

TL;DR: VSCode settings that affect how other developers working on your code will commit seem bad to me, unless there is an easy and accepted way to get these settings into the git repository of the code base.

**Comment by EddieDover** [comment]
I agree this is something that could and should be looked into. For now, would an acceptable compromise me bumping the default line length in the package.json to match that of the toolkit?

**Comment by chrisl8** [comment]
Ideally, in my mind, the extension would not pass any line length at all unless one was "set" in the settings. It seems like the current code always sets something.

Yes, setting the default in the extension to match the default from `gdformat` would cover for the issue, but seems like a "hack".
Practically though, yes, that would do the trick.

I've also discovered now that many people suggest committing the `.vscode` folder into a repository, so maybe that is the simple answer, to set it in the Workspace area and commit it.

**Comment by EddieDover** [comment]
Again, I agree with you but I won't have any bandwidth to personally look into it until next weekend. If you'd like to take a wack at it, I'd welcome a PR.

**Comment by EddieDover** [comment]
> I've also discovered now that many people suggest committing the `.vscode` folder into a repository, so maybe that is the simple answer, to set it in the Workspace area and commit it.

Honestly, this is the solution I'd go with. The extension will read modifications in the settings.json, so setting it via the workspace is probably the best bet. I've always committed my .vscode folder.

However, if we've decided this isn't a good enough solution by next weekend, then I will change things so that the config priority is the gdlint config if it exists then the workspace settings.

**Comment by chrisl8** [comment]
Thanks! If I decide I need anything fancier myself I'll make a PR.
